src/redux/actions/samples/updateSampleFile.js

```diff
@@ -16,18 +17,22 @@ const updateSampleFile = (
 ) => async (dispatch, getState) => {
   const updatedAt = moment().toISOString();
   const sample = getState().samples[sampleUuid];
+  // we'll need to remove the hard-coded 10x tech type once we start
+  // supporting other types and save the chosen tech type in redux
+  const valid = checkIfFileValid(fileName, '10X Chromium');
```

Comment:
we are using `updateSampleFile` only in `processUpload` and usually not to create a new sample file but to update an existing one so it is usually not necessary to do this check, I assume this validity check was added here so that it is run when we are using `UploadDetailsModal` to update a file, could it be moved into that modal instead then?

Reply:
But we still need to provide a `valid` field here so that the analysis can be launched. Do you think it should be just valid: true here, while the check is done in the `UploadDetailsModal`?

Reply:
Yes, I believe that's the pattern we are already following in FileUploadModal